roles/docker/tasks/cleanup-rpm.yml

```diff
@@ -0,0 +1,14 @@
+- name: install docker-cleanup package
+  sudo: yes
+  yum:
+    name: "https://dl.bintray.com/asteris/mantl-rpm/docker-cleanup-0.0.1-1.noarch.rpm"
+    state: present
+  tags:
+    - docker
+
+- name: exclude zookeeper data volume from garbage collection
+  sudo: yes
+  lineinfile:
+    dest: /etc/docker-cleanup/docker-gc-exclude-containers
```

Comment:
This should be an ansible var that can contain multiple values, so that users can configure volumes to exclude.